src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -264,6 +264,9 @@
     text: |
         az acr import -n MyRegistry --source sourcerepository:sourcetag -t targetrepository:targettag \\
             -r /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sourceResourceGroup/providers/Microsoft.ContainerRegistry/registries/sourceRegistry
+  - name: Import an image and return right after starting an import.
```

Comment:
Maybe reword in a way like....
```suggestion
  - name: Import an image without waiting for completion.
```
or 
```suggestion
  - name: Import an image. Do not wait for completion.
```
Just some suggestions to consider.

Reply:
Also you can update the short summary above to add:
```
For large images consider using `--no-wait`.
```